scripts/popup.js

```diff
@@ -300,6 +300,15 @@ function settings() {
   $('#setting-page').show()
 }
 
+document.getElementById('webstore-url').style.display = 'none';
+function share(webstoreURL) {
+  let $temp = $('<input>')
+  $('body').append($temp)
+  $temp.val($(webstoreURL).text()).select()
+  document.execCommand('copy')
```

Comment:
Have you tested it in firefox?

Reply:
It doesn't works for firefox. I am searching for a fix, will update when I find it.